tests/plotting/test_theme.py

```diff
@@ -388,6 +388,21 @@ def test_theme_slots(default_theme):
     with pytest.raises(AttributeError, match='has no attribute'):
         default_theme.new_attr = 1
 
+    # verify we can't create an arbitrary attribute on an attribute
+    assert default_theme.lighting_params
+    with pytest.raises(AttributeError, match='has no attribute'):
+        default_theme.lighting_params.new_attr = 1
+
+    # Repeat with a subclass
+    theme = DarkTheme()
+    with pytest.raises(AttributeError, match='has no attribute'):
+        theme.new_attr = 1
+
+    # verify we can't create an arbitrary attribute on an attribute
+    assert default_theme.lighting_params
+    with pytest.raises(AttributeError, match='has no attribute'):
+        default_theme.lighting_params.new_attr = 1
```

Comment:
We need a comment to why this test is being run again after defining the subclass.

Reply:
Good catch, this was supposed to test the subclass.  Now fixed.